MiniApp/Classes/Extensions/MiniApp+NSError.swift

```diff
@@ -30,4 +30,12 @@ extension NSError {
             userInfo: [NSLocalizedDescriptionKey: "Invalid response received"]
         )
     }
+
+    class func downloadingFailed() -> NSError {
```

Comment:
Does Android use this naming? I think `downloadFailed` and same naming in error message would be better.

Reply:
I couldn't find anything related to this in Android side.

Reply:
Android uses synchronous `suspend` functions, so it doesn't have anything similar to this.